pages/deployment/github-actions.mdx

```diff
@@ -94,3 +94,13 @@ depending on whether you want the website to be deployed via that protocol.
 The `refresh_token` should contain a Github secret that you set up earlier, 
 and the `site_url` field should contain the subdomain you set up to host your website. 
 
+## Powered By Distributed Press
+
+We'd appreciate if you embedded the following HTML snippit on your DP-published sites to help spread the word.
+What's cool is it will load the site over the same protocol as the one used to access your site.
+
+```html
+<p style="font-family: Source Sans Pro; font-size: 0.8em;">
+ <a href="//distributed.press">Powered By Distributed Press</a>
+</p>
+```
```

Comment:
```html
<p style="font-family: Source Sans Pro; font-size: 0.8em;">
 <a href="//distributed.press">Powered By Distributed Press</a>
</p>
```
```suggestion
  <!-- Published with Distributed Press Snippet -->
  <div
    style="
      font-family: Arial, sans-serif;
      text-align: center;
      margin-top: 20px;
      padding: 8px;
    "
  >
    <!-- Powered By Text -->
    <div style="font-size: 12px; color: #333; margin-bottom: 10px">
      Powered by
    </div>

    <!-- Button Link -->
    <a
      href="//distributed.press"
      target="_blank"
      style="
        display: inline-block;
        width: 90px;
        height: 31px;
        background-color: #d3d3d3; /* Retro gray background */
        border: 1px solid #808080; /* Slightly darker gray border */
        text-align: center;
        line-height: 31px;
        font-size: 12px;
        font-family: 'Old English Text MT', 'Cloister Black', 'Engravers Old English', 'Blackletter 686', serif;
        color: #000; /* Black text */
        text-decoration: none;
        text-shadow: 1px 1px 0 #fff; /* Light retro text shadow */
        box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.2); /* Slight retro button shadow */
      "
    >
      Distributed Press
    </a>

    <!-- Links Below Button -->
    <p style="font-size: 12px; color: #555; margin: 8px 0 0">
      <a
        href="//reader.distributed.press"
        target="_blank"
        style="color: #0078d7; text-decoration: none"
        >Follow on ActivityPub</a
      >
      |
      <a
        href="//docs.distributed.press"
        target="_blank"
        style="color: #0078d7; text-decoration: none"
        >Learn More</a
      >
    </p>
  </div>
```

How about this? we can remove the reader and docs part, the button looks cool.
![Screenshot 2024-11-27 at 2 50 11 PM](https://github.com/user-attachments/assets/0fb54139-ca79-42d6-b657-ee6eba3cb9bb)

Reply:
I like the extra links but I think the styling is too much and doesn't match the rest of DP. Remove all the button styles and have an img tag pointing to our logo on the site: 

```html
<img alt="Distributed Press" src="https://distributed.press/img/logos/logo-distributedpress-grey.png"/>
```

Reply:
Also lets remove all font/text styling so it gets inherited from the page it's ojn instead.